src/Umbraco.Core/Constants-PropertyEditors.cs

```diff
@@ -42,10 +42,14 @@ public static class PropertyEditors
             [Obsolete("GUIDs are no longer used to reference Property Editors, use the Alias constant instead. This will be removed in future versions")]
             public const string ContentPicker = "158AA029-24ED-4948-939E-C3DA209E5FBA";
 
+            
+            [Obsolete("This is an obsoleted content picker, use ContentPicker2Alias instead")]
+            public const string ContentPickerAlias = "Umbraco.ContentPickerAlias";
+
             /// <summary>
             /// Alias for the Content Picker datatype.
             /// </summary>
-            public const string ContentPickerAlias = "Umbraco.ContentPickerAlias";
+            public const string ContentPicker2Alias = "Umbraco.ContentPicker2";

```

Comment:
@warrenbuckley we'll just leave it for now, otherwise feel free to just change it.